pkg/build/strategies/strategies.go

```diff
@@ -27,6 +27,19 @@ func Strategy(client docker.Client, config *api.Config, overrides build.Override
 	fs := fs.NewFileSystem()
 
 	startTime := time.Now()
+
+	assembleUser, err := docker.GetAssembleUser(config.AssembleUser, config.BuilderImage, client, config)
```

Comment:
Would this work here?
config.AssembleUser, err := docker.GetAssembleUser(config.AssembleUser, config.BuilderImage, client, config)